Benchmarking tool fails to build with NetCDF-only builds of Scorpio

The benchmarking tool, pioperf_rearr, fails to build with NetCDF-only builds of Scorpio with the following error message (make tests),
```
/sandbox/jayesh/tmp/PIOTest/tests/performance/pioperformance_rearr.F90:1049:72:

        iostat = PIO_put_att(File, varr(nv), "_FillValue", PIO_FILL_FLOAT);
                                                                        1
Error: Symbol ‘pio_fill_float’ at (1) has no IMPLICIT type
/sandbox/jayesh/tmp/PIOTest/tests/performance/pioperformance_rearr.F90:1038:51:

    iostat = PIO_def_dim(File, 'time', PIO_UNLIMITED, dimid(ndims+1))
                                                   1
Error: Symbol ‘pio_unlimited’ at (1) has no IMPLICIT type
/sandbox/jayesh/tmp/PIOTest/tests/performance/pioperformance_rearr.F90:682:29:

        dfld = PIO_FILL_DOUBLE
                             1
Error: Symbol ‘pio_fill_double’ at (1) has no IMPLICIT type
/sandbox/jayesh/tmp/PIOTest/tests/performance/pioperformance_rearr.F90:681:28:

        rfld = PIO_FILL_FLOAT
                            1
Error: Symbol ‘pio_fill_float’ at (1) has no IMPLICIT type
/sandbox/jayesh/tmp/PIOTest/tests/performance/pioperformance_rearr.F90:680:26:

        ifld = PIO_FILL_INT
                          1
Error: Symbol ‘pio_fill_int’ at (1) has no IMPLICIT type
make[3]: *** [tests/performance/CMakeFiles/pioperf_rearr.dir/pioperformance_rearr.F90.o] Error 1
make[3]: Leaving directory `/sandbox/jayesh/tmp/PIOTest/build'
make[2]: *** [tests/performance/CMakeFiles/pioperf_rearr.dir/all] Error 2
make[2]: Leaving directory `/sandbox/jayesh/tmp/PIOTest/build'
make[1]: *** [CMakeFiles/tests.dir/rule] Error 2
make[1]: Leaving directory `/sandbox/jayesh/tmp/PIOTest/build'
make: *** [tests] Error 2

```
@dqwu noticed this issue after merging PR #271 to master. Also this issue occurs only for shared library builds (does not occur for static library builds)

--- a/src/flib/CMakeLists.txt
+++ b/src/flib/CMakeLists.txt
@@ -194,7 +194,7 @@ endif ()
 if (WITH_NETCDF)
   find_package (NetCDF ${NETCDF_FORTRAN_MIN_VER_REQD} COMPONENTS Fortran)
 endif ()
-if (NetCDF_FOUND)
+if (NetCDF_Fortran_FOUND)
   target_include_directories (piof
     PUBLIC ${NetCDF_Fortran_INCLUDE_DIRS})
   target_compile_definitions (piof

